plugins/org.locationtech.udig.libs/META-INF/MANIFEST.MF

```diff
@@ -90,6 +89,7 @@ Import-Package: oracle.core.lmx;resolution:=optional,
  javax.media.jai.tilecodec,
  javax.media.jai.util,
  javax.media.jai.widget,
+ org.apache.commons.codec;version="1.9.0",

```

Comment:
I'd suggest o add other exported packages from commons-codec as well, otherwise it might fail to NoClassDefFoundErrors

```
org.apache.commons.codec;version="1.2.0"
org.apache.commons.codec.binary;version="1.2.0"
org.apache.commons.codec.digest;version="1.2.0",
org.apache.commons.codec.language;version="1.2.0"
org.apache.commons.codec.language.bm;version="1.2.0"
org.apache.commons.codec.net;version="1.2.0"
```

and I'd prefare the version that is the minimun required version which is:

from dependency-tree, if common-codec is not managed as provided in pom-libs:

```
[INFO] +- org.geotools:gt-wms:jar:22.1:compile
[INFO] |  \- commons-httpclient:commons-httpclient:jar:3.1:compile
[INFO] |     \- commons-codec:commons-codec:jar:1.2:compile
```